### PR TITLE
Lesson3 new

### PR DESCRIPTION
**Pull request by alex-rns**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    /**
+     * add custom template
+     */
+    protected function _prepareLayout()
+    {
+        parent::_prepareLayout();
+        $this->setTemplate(self::LESSON3_TEMPLATE);
+        return $this;
+    }
+    public function getUrlToJsonResponse() {
```
Comment (chabanenk0) [inline app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php]: Please add new lines between the methods. Move the open curly brace to the next line, Please use phpdocs (phpstorm can generate it)

File: app/code/BelodubrovskyiAn/MyCustomModule/Controller/ShowPerson/Index.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+     * @return \Magento\Framework\App\ResponseInterface|\Magento\Framework\Controller\ResultInterface|void
+     */
+    public function execute()
+    {
+        $name = "Aleksandr";
+        $lastName = "Belodubrovskyi";
+        /** @var \Magento\Framework\View\Result\Page $resultPage */
+        $resultPage = $this->resultFactory->create(ResultFactory::TYPE_PAGE);
+        $resultPage->getLayout()->getBlock('homework.custom.block')->setName($name)->setLastName($lastName);
+        return $resultPage;
```
Comment (chabanenk0) [inline app/code/BelodubrovskyiAn/MyCustomModule/Controller/ShowPerson/Index.php:20]: Please add one empty new line before return

File: app/code/BelodubrovskyiAn/MyCustomModule/etc/frontend/routes.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
+    <!--Use router 'standard' for frontend route-->
+    <!--Use router 'admin' for admin route, AND PUT THIS file to etc/adminhtml -->
```
Comment (chabanenk0) [inline app/code/BelodubrovskyiAn/MyCustomModule/etc/frontend/routes.xml]: Remove these comments

File: app/code/BelodubrovskyiAn/MyCustomModule/view/frontend/templates/lesson/myhomework.phtml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?php
+/** @var \Magento\Framework\View\Element\Template $block */
+?>
+<div class="content-wrapper">
+    <h1>
+        <span>My homework was successfully</span>
```
Comment (chabanenk0) [inline app/code/BelodubrovskyiAn/MyCustomModule/view/frontend/templates/lesson/myhomework.phtml]: use translation here

File: app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <?= $block->getChildHtml( 'title_text');?>
```
Comment (chabanenk0) [inline app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml]: no need the space before the argument:
$block->getChildHtml( 'title_text');  
correct to 
$block->getChildHtml('title_text');

File: app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php (hunk 1 above)
Comment (alex-rns) [reply app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php]: fix

Comment (alex-rns) [reply app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php]: fixed

File: app/code/BelodubrovskyiAn/MyCustomModule/Controller/ShowPerson/Index.php (hunk 2 above)
Comment (alex-rns) [reply app/code/BelodubrovskyiAn/MyCustomModule/Controller/ShowPerson/Index.php:20]: fixed

File: app/code/BelodubrovskyiAn/MyCustomModule/etc/frontend/routes.xml (hunk 3 above)
Comment (alex-rns) [reply app/code/BelodubrovskyiAn/MyCustomModule/etc/frontend/routes.xml]: fixed

File: app/code/BelodubrovskyiAn/MyCustomModule/view/frontend/templates/lesson/myhomework.phtml (hunk 4 above)
Comment (alex-rns) [reply app/code/BelodubrovskyiAn/MyCustomModule/view/frontend/templates/lesson/myhomework.phtml]: Use)

File: app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml (hunk 5 above)
Comment (alex-rns) [reply app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml]: Fixed

**Pull request merged**